server-spi/src/main/java/org/keycloak/models/UserModel.java

```diff
@@ -311,7 +311,9 @@ enum RequiredAction {
         UPDATE_PASSWORD,
         TERMS_AND_CONDITIONS,
         VERIFY_PROFILE,
-        UPDATE_EMAIL
+        UPDATE_EMAIL,
+        WEBAUTHN_REGISTER,
```

Comment:
Maybe there is no need to add these required actions to UserModel at all as they are not used anyway due the backwards compatibility (besides potentially incorrect usage, which I've pointed in the comments above). I think we already have few other required actions, which are not on UserModel. My vote is to revert this change and keep the UserModel class unchanged in this PR.

Reply:
Agree.